node/src/node_configurator/unprivileged_parse_args_configuration.rs

```diff
@@ -1537,33 +1534,7 @@ mod tests {
                     .unwrap()
             )),
         );
-        assert_eq!(
```

Comment:
This assert should probably not be removed, but should appear somewhere else where the neighborhood mode is Standard. It protects against Standard-mode sloppiness, and we still need that protection in Standard mode.